wasm/Cargo.toml

```diff
@@ -82,12 +83,21 @@ version = "0.1.7"
 [dependencies.rand_chacha]
 version = "0.3.1"
 
+[dependencies.wasm-bindgen-rayon]
+version = "1.0"
+
+[dependencies.rayon]
+version = "1.5"
+
 [dev-dependencies.wasm-bindgen-test]
 version = "0.3.33"
 
 [profile.release]
 opt-level = 3
 lto = true
 
+[package.metadata.wasm-pack.profile.debug]
+wasm-opt = ["-O4"]
```

Comment:
Important note:
These build optimizations are extremely important when locally debugging as without them my proofs were so slow that they were basically non-functional.